### PR TITLE
Fixes #943 Adding Google Analytics

### LINKED ISSUE #943: Quickstart should include Google Analytics
## Motivation
Google Analytics is the standard way that we track web analytics.

### Is your feature request related to a problem? Please describe.
We need to have a solution for analytics available at launch.

### Describe the solution you'd like

- Google Tag Manager enabled by default with the same Container ID (GTM-ML2BZB) as QS1 included. 
- Google Analytics included, but disabled by default (to allow for departments to add their own property id)

### Roles and Permissions considerations
A clear and concise description of how each of the following roles would be impacted by this change:
- Anonymous user
- Authenticated user
- Content editor
- Content administrator
- Administrator

### PR DESCRIPTION
**Pull request by camikazegreen**
## Description
I added Google Analytics to composer.json

## Related Issue
#943 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
         "drupal/externalauth": "1.4.0",
         "drupal/field_group": "3.1",
         "drupal/field_group_link": "3.0-rc2",
+        "drupal/google_analytics:2.5",
```
Comment (camikazegreen) [inline composer.json:63]:
```suggestion
        "drupal/google_analytics":"2.5",
```

**Comment by trackleft** [comment]
Is it weird that a release labelled 3.1 is yellow? https://www.drupal.org/project/google_analytics

**Comment by joeparsons** [comment]
> Is it weird that a release labelled 3.1 is yellow? https://www.drupal.org/project/google_analytics

From the [8.x-3.0-beta1 release notes](https://www.drupal.org/project/google_analytics/releases/8.x-3.0-beta1):
>Keep in mind that there are features missing in Google gtag.js API (8.x-3.x) that exists in Google analytics.js API (8.x-2.x). Therefore if you have advanced features in use you may need to stay at Google Analytics 8.x-2.x until these missing features come back.

**Comment by trackleft** [comment]
@cgustafson Says that you can add GTM and UA via Google gtag 

Here is some documentation for which to choose. https://support.google.com/tagmanager/answer/7582054?hl=en

**Comment by trackleft** [comment]
I think this PR is fine as long as we also include GTM since that is where our business analysts spend most of their time, and rarely need to add custom javascript events that aren't possible within the GTM UI.

**Review by smayden — APPROVED** [review]
Not certain if more discussion is needed around which version to use, but I tested this on the Probo site:
- Enabled module
- Added GA property to Google Analytics configuration
- Verified that Google Analytics tracking code and property ID appear in page source for pages on site
- Verified that activity on probo site appearing in Google Analytics for the associated property
- Disabled Google Analytics module

**Review by joeparsons — APPROVED** [review]

**Comment by camikazegreen** [comment]
We can use Google Tag Manager to include a second tag that includes Google Analytics. That makes this module redundant.

**Pull request closed without merging**